app/views/notices/_works_urls.html.erb

```diff
@@ -2,8 +2,11 @@
   <div class="row">
     <span class="label original-title"><%= original_title %></span>
     <ol class="list original-urls">
-      <%# Don't do code style changes here, it's intentional to avoid additional spaces in the response %>
-      <% work.copyrighted_urls.each do |url| %><%= content_tag_for(:li, url) do %><%= url.url %><% end %><% end %>
+      <% if work.copyrighted_urls.each do |url| %>
+        <%= content_tag_for(:li, url) do %><%= url.url %><% end %>
+      <% end.empty? %>
+        No copyrighted URLs were submitted.
```

Comment:
This doesn't work, can't see this warning on the front-end.

Reply:
![Screenshot from 2019-03-14 18-55-38](https://user-images.githubusercontent.com/2022788/54379997-cb321900-468a-11e9-8150-befb73c61d6b.png)

Reply:
That example seems to be missing an entire section that I see on a notice of the same type locally:
<img width="902" alt="Screen Shot 2019-03-21 at 2 05 47 PM" src="https://user-images.githubusercontent.com/959761/54774586-7dfdfc00-4be2-11e9-9ca4-651058f671e8.png">

Note the `Enforcement Request #1` section that is in my screenshot but not yours. That maps to a `Work` that is associated with the `Notice`. The part with copyrighted/infringing URLs is a child of the template that renders works. Since this example has no works, we never get there.

By contrast note this, after I have deleted the associated URLs:

<img width="686" alt="Screen Shot 2019-03-21 at 2 12 28 PM" src="https://user-images.githubusercontent.com/959761/54775015-68d59d00-4be3-11e9-9d3d-e964e0e0863f.png">

Which I guess is a question about scope. Do you think the intent of this ticket is that users should see a warning also when there are no *works* attached to the notice? Or that that belongs in a different ticket, possibly with different language?

Reply:
Yeah, I was disconnecting works, not work urls. This looks fine.